File: Week 6/Sentimental-Cash/cash.py
x = int(input("Change owed: "))
coins = 1
quarter = 25
ten_cent = 10
five_cent = 5
cent = 1

if x > quarter:
    while x > quarter:
        x -= quarter
        coins += 1

if x > ten_cent:
    while x > ten_cent:
        x -= ten_cent
        coins += 1

if x > five_cent:
    while x > five_cent:
        x -= five_cent
        coins += 1

if x > cent:
    while x > cent:
        x -= cent
        coins += 1

print(coins)

File: Week 1/Cash/cash.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // Declaring variables.
    int change = 0;
    int cent[] = {25, 10, 5, 1};
    int coins = 0;

    // Prompts the user for the change owed.
    do
    {
        change = get_int("Change owed: ");
    }
    while (change < 0);
    // Calculates the amount of coins to be paid.
    int i = 0;
    while (change != 0)
    {
        while (change >= cent[i])
        {
            change = change - cent[i];
            coins++;
        }
        i++;
    }

    // Outputs the number of coins to be used in returning change.
    printf("Coins: %d\n", coins);
}
